qiskit/providers/aer/noise/errors/quantum_error.py

```diff
@@ -187,35 +187,37 @@ def copy(self):
 
     @property
     def atol(self):
-        """The absolute tolerance parameter for float comparisons."""
-        return self.ATOL
-
-    @atol.setter
-    def atol(self, atol):
-        """Set the absolute tolerance parameter for float comparisons."""
-        max_tol = self.MAX_TOL
-        if atol < 0:
-            raise NoiseError("Invalid atol: must be non-negative.")
-        if atol > max_tol:
-            raise NoiseError(
-                "Invalid atol: must be less than {}.".format(max_tol))
-        self.ATOL = atol
+        """The default absolute tolerance parameter for float comparisons."""
+        return QuantumError._ATOL_DEFAULT
 
     @property
     def rtol(self):
         """The relative tolerance parameter for float comparisons."""
-        return self.RTOL
-
-    @rtol.setter
-    def rtol(self, rtol):
-        """Set the relative tolerance parameter for float comparisons."""
-        max_tol = self.MAX_TOL
-        if rtol < 0:
-            raise NoiseError("Invalid rtol: must be non-negative.")
-        if rtol > max_tol:
+        return QuantumError._RTOL_DEFAULT
+
+    @classmethod
+    def set_atol(cls, value):
+        """Set the class default absolute tolerance parameter for float comparisons."""
```

Comment:
I see the logic for tolerances follows the `base_operator` class in `terra`, but I find a bit weird to change class attributes through set methods in instantiated objects. Maybe a meta class would have been the way to go here.
This is just a comment, let's not halt the merge because of this.

Reply:
The idea was you call these on the class `QuantumError.set_atol(1e-7)` (of course since its python nothing stops you calling it on an object instance too)

Reply:
Yes, you are right. I expressed myself quite bad.  My concern was why not use the `property` mechanism for the class members themselves through a metaclass, so that you do `QuantumError.atol = 1e-7` instead of `QuantumError.set_atol(1e-7)`. That way you can even avoid   the possibility of setting `atol` from an instance or mistakenly do `QuantumError.atol = 1e-7`, creating a new class variable `atol` and leaving `_ATOL_DEFAULT` unchanged.
But yes, it's Python and "_We are consenting adults_" here  :)
Jokes aside, I'm ok with this.

Reply:
Can you make an issue in terra about how to do this?